kyaml/kio/byteio_reader.go

```diff
@@ -275,6 +294,14 @@ func (r *ByteReader) decode(originalYAML string, index int, decoder *yaml.Decode
 	// sort the annotations by key so the output Resources is consistent (otherwise the
 	// annotations will be in a random order)
 	n := yaml.NewRNode(node)
+	wrappedNode := yaml.NewRNode(&yaml.Node{
```

Comment:
nit: I would suggest creating wrappedNode inside the `if` block where we detect bareSequenceNode and then overwriting the `n` with wrappedNode. Using `n` in the rest of the code.
Basically, isolating the wrapping functionality where detected.

Reply:
Updated.